core/app/models/spree/order_updater.rb

```diff
@@ -221,21 +208,17 @@ def update_cancellations
 
     def update_item_totals
       [*line_items, *shipments].each do |item|
-        # The cancellation_total isn't persisted anywhere but is included in
-        # the adjustment_total
-        item.adjustment_total = item.adjustments.
-          reject(&:included?).
-          sum(&:amount)
-
-        if item.changed?
-          item.update_columns(
-            promo_total:          item.promo_total,
-            included_tax_total:   item.included_tax_total,
-            additional_tax_total: item.additional_tax_total,
-            adjustment_total:     item.adjustment_total,
-            updated_at:           Time.current,
-          )
-        end
+        Spree::ItemTotal.new(item).recalculate!
```

Comment:
Since we make this a class now, can we also make this configurable, so that we have one less potential patch for users?

Reply:
Definitely! I don't see any harm in doing so.